n_serial.c

```diff
@@ -59,18 +49,18 @@ const char *serialNoteTransaction(char *json, char **jsonResponse)
         }
     }
 
-    // Free the transmit buffer
-    _Free(transmitBuf);
+    // Append newline to the transaction
+    _SerialTransmit((uint8_t *)"\r\n", 2, true);

```

Comment:
Should we use `c_newline` and `c_newline_len` here? Not critical, but the constants are there, so might as well use them.

Reply:
yes we should